lib/elixir/pages/Syntax Reference.md

```diff
@@ -485,7 +485,7 @@ You can see them being used in constructs such as `receive`, `try`, and others.
 
 This document provides a quick reference to Elixir syntax, exploring the simplicity behind its AST and documenting the base constructs with their AST equivalents.
 
-We have also discussed a handful of syntax conveniences provided by Elixir. Those conveniences are what allow us to write
+We have also discussed a handful of syntax conveniences provided by Elixir. Those conveniences enable us to write
```

Comment:
I'm not sure the shade of this is the same as before; we're saying that we explained the syntax conveniences because they are what enables us to write the code below. Wdyt?